examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.cpp

```diff
@@ -545,11 +545,11 @@ sl_status_t show_scan_results(sl_wifi_scan_result_t * scan_result)
 
         cur_scan_result.ssid_length = strnlen((char *) scan_result->scan_info[idx].ssid,
                                               chip::min<size_t>(sizeof(scan_result->scan_info[idx].ssid), WFX_MAX_SSID_LENGTH));
-        chip::Platform::CopyString(cur_scan_result.ssid, cur_scan_result.ssid_length, (char *) scan_result->scan_info[idx].ssid);
+        chip::Platform::CopyString(cur_scan_result.ssid, cur_scan_result.ssid_length + 1, (char *) scan_result->scan_info[idx].ssid); // +1 for null termination
```

Comment:
So I didn't notice in my first review but here and in the other places, what is the ssid destination buffer size? From what I can see in wfx_wifi_scan_result struct, ssid buffer is
`char ssid[WFX_MAX_SSID_LENGTH + 1];` if this is always the case then we are good and this is safe.

but if some ssid destination buffer are of size `WFX_MAX_SSID_LENGTH`, and if
`cur_scan_result.ssid_length` is `WFX_MAX_SSID_LENGTH` because of the chip::min at line 546.

the +1 would cause the null termination to be written out of bounds.

Here we seem safe. I'll suggest a comment addition to let other working on that file that the null space is account for